### ISSUE TITLE
Explain where polling data comes from 

### ISSUE THREAD

**Issue by aspiers** [swapmyvote/swapmyvote#907]
Currently we just display the bar charts and interpretations without explaining where they come from.  We need:

- [ ] A help icon / tooltip thing next to the charts and/or interpretations so it's clear how to find out more.
- [ ] A FAQ entry which the above can link to.

**Comment by aspiers** [comment]
@tdg Does this sound reasonable?

**Comment by baob** [comment]
Pretty sure @tdg asked me for this info not long ago. He may be working on something already.

**Comment by tdg** [comment]
Yes we have discussed this and I suggest lifting quite a bit of the copy and framing from our cousins' site who have helpfully aggregated the polls for us

**Comment by aspiers** [comment]
Yeah, @baob please can you summarise how we obtained the STT polling data?  Is it one of the files at https://stopthetories.vote/data ?

**Comment by aspiers** [comment]
If they've documented the process we could just link to that page or wherever.

**Comment by baob** [comment]
So far as I know, it's not public. Comments in the code say
```
updates can be obtained by exporting the tab "MRPs Calc" as a csv from the shared google spreadsheet
```
and even that's not quite accurate. I think I used a neighbouring tab in the google sheet.
